tests/integ/sagemaker/jumpstart/retrieve_uri/constants.py

```diff
@@ -0,0 +1,118 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+from enum import Enum
+from typing import Dict
+from typing import Optional
+from typing import Union
+import os
+
+
+def _to_s3_path(filename: str, s3_folder: Optional[str]) -> str:
+    return filename if not s3_folder else f"{s3_folder}/{filename}"
+
+
+_NB_ASSETS_S3_FOLDER = "inference-notebook-assets"
+_TF_FLOWERS_S3_FOLDER = "training-datasets/tf_flowers"
+
+TMP_DIRECTORY_PATH = os.path.join(
+    os.path.abspath(os.path.join(os.path.abspath(__file__), os.pardir)), "tmp"
+)
+
+ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID = "JUMPSTART_SDK_TEST_SUITE_ID"
+
+JUMPSTART_TAG = "JumpStart-SDK-Integ-Test-Suite-Id"
+
+HYPERPARAMETER_MODEL_DICT = {
+    ("huggingface-spc-bert-base-cased", "*"): {
+        "epochs": "1",
+        "adam-learning-rate": "2e-05",
+        "batch-size": "8",
+        "sagemaker_submit_directory": "/opt/ml/input/data/code/sourcedir.tar.gz",
+        "sagemaker_program": "transfer_learning.py",
+        "sagemaker_container_log_level": "20",
+    },
+}
+
+TRAINING_DATASET_MODEL_DICT = {
+    ("huggingface-spc-bert-base-cased", "*"): ("training-datasets/QNLI/"),
+}
```

Comment:
you should use tiny datasets. Please refer to modelhub integ tests dataset [keys](https://github.com/aws/sagemaker-model-hub/blob/c0c6636b7d9e8fc4f722f340af280b7b388412c9/src/sagemakermodelhub/integ_tests/test_engine/specs/spec_override.py#L9).